gateway/enforcer/org.wso2.apk.enforcer/src/main/java/org/wso2/apk/enforcer/subscription/SubscriptionDataStoreImpl.java

```diff
@@ -278,113 +246,6 @@ public void addApplicationKeyMappings(
         this.applicationKeyMappingMap = newApplicationKeyMappingMap;
     }
 
-    @Override
-    public void addOrUpdateSubscription(Subscription subscription) {
```

Comment:
we might need these methods in subscription validation scenarios, maybe we can remove policy related methods and let's keep subscription/application related methods